service/src/test/java/bio/terra/workspace/db/ResourceDaoTest.java

```diff
@@ -51,6 +60,24 @@ public void setUp() {
     workspaceUuid = createWorkspaceWithGcpContext(workspaceDao);
   }
 
+  @BeforeEach
+  public void setUpMocks() throws IOException {
```

Comment:
rm? I don't see any code changes in ResourceDao.

Reply:
Hi Yu. We need that because when we get the AinotebookInstance, they will get the cpu and gpu information of the instance. the instances.get API needs a mock here. See the screenshot when i remove the mock.
https://screenshot.googleplex.com/4wRToMauJrMq3os

I also feel a little bit strange but seems like this is the easiest way. let me know if you have better approaches.

Reply:
oic the handler.

@beforeAll.

Reply:
I also tried to use the BeforeAll, but also fails like the screenshot. (Sorry, I should mentioned it in the last comment, I forget...) It seems like the mock will expire after each unitTest finish.